utils/docker-compose/README.md

```diff
@@ -0,0 +1,227 @@
+# FPC Docker Compose Network
+This docker-compose example has been adapted from a [Fabric 101 Workshop](https://github.com/swetharepakula/Fabric101Workshop) which was adapted from the basic
+network and fabcar example in the [Fabric Samples](https://github.com/hyperledger/fabric-samples).
+This example does not use TLS, which means the Fabric Go SDK cannot be
+used to interact with the network. Currently, there are two orgs, one peer,
+one orderer, and one fabric-ca in the network.
+
+## Configuration
+- [core-fpc.yaml](network-config/core-fpc.yaml) : Peer configuration that has
+the SGX plugins and locations relative to location within docker image
+- [core.yaml](network-config/core.yaml) : Regular Peer configuration
+without FPC. Used if `$USE_FPC` is set `false`.
+- [orderer.yaml](network-config/orderer.yaml) : Orderer configuration
+- [crypto-config.yaml](network-config/crypto-config.yaml) : File used with cryptogen to generate
+certs for specified number of orgs, peers, users, and orderer. The CA credentials
+can be used to start instances of fabric-ca
+- [configtx.yaml](network-config/configtx.yaml)  : File used with configtxgen to generate the
+genesis block which is used as the basis of the specified channel
+- [docker-compose.yml](network-config/docker-compose.yml) : Configuration of the
+fabric network to be used with `docker-compose`. This file depends on two
+environment variables to properly bring up a network. `$FPC_CFG` can be set to
+`-fpc` or shall be empty. If set to `-fpc` the `core-fpc.yaml` & FPC peer image
+is used. Otherwise it will use `core.yaml` and the regular peer image.
+`$PEER_CMD` must also be set to the location of binary or script that will start
+ the peer.  **Docker version 17.06.2-ce or higher is needed**
+
+## Steps
+1. Download the necessary fabric binaries. Run the
+[bootstrap script](scripts/bootstrap.sh) which will download the Fabric 1.4.3
+into a local bin directory. If you already have the binaries downloaded in your
+path, this step can be skipped. **Fabric 1.4.3 versions of configtxgen and
+cryptogen are required to use the configurations above.**
+```
+./scripts/bootstrap.sh
+```
+1. Build the peer image in `utils/docker/peer` directory which is defined by the
+peer [Dockerfile](../docker/peer/Dockerfile). This step
+assumes you have already built the [fabric-private-chaincode base image](../docker/base/Dockerfile).
```

Comment:
Given that we have makefiles to build the other images, would it make sense to add a make target also for the peer? If so, of course, the question would be which config to use for which i would argue the local checkoued repo version (file:///tmp/build-src/.git  variant) might be the natural one?

Reply:
I do like this. I am envisioning `peer` and a `peer-local` target. For now I am going to add the `peer` target only and in a followup I will add the local one.